stack/keycloak/migration/01_init-realm.yml

```
@@ -1,9 +1,10 @@
+id: init realm
 author: Default
 changes:
   - addRealm:
       name: local_realm
 
   - addClientScope:
       realm: local_realm
-      name: openid
\ No newline at end of file
+      name: openid
+      description: Client scope used for OpenID Connect (OIDC) based authentication
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)